### DISCUSSION TITLE
Toggle column in table, cannot be used row model object.

### DISCUSSION THREAD

**Discussion by arslonbeksaidov** [filamentphp/filament#13550] [Ideas]
Tables\Columns\ToggleColumn::make('transfer_can_download')
                    ->hidden(function (Application $application): bool {
                            **dd($application); // Null showed insted application object**
//                        if ($application->detail->transferType->semester != Semester::FIRST){
//                            return true;
//                        }
                        return BasicAuth::user()->can('update_transfer_can_download');
                    })
![image](https://github.com/user-attachments/assets/37c26779-33b7-4b70-a2c6-8bdc658e746e)

**Comment by leandrocfe** [comment]
You can't show/hide a column by row. You should use `disabled()` instead of this
